app/components/NotificationModal.vue

```diff
@@ -1,52 +1,119 @@
+                <GridLayout v-for="n in dismissedNotifications" :key="n.id">
+                    <GridLayout rows="*,*,*" columns="15*,85*" class="m-x-10 notify-box size-14">
+                        <Image col="0" row="0" src="~/images/Icon_Archive_Bell.png" width="21" />
+                        <Label col="1" row="0" :text="n.heading" textWrap="true" class="bold"  @loaded="onLabelLoadedVerticalCentered"/>
+                        <Label col="1" row="1" :text="n.text" textWrap="true" lineHeight="4"/>
+                        <GridLayout col="1" row="2" columns="auto,auto" class="size-12 bold" :dataId="n.id">
+                            <Label col="0" :text="'Add Field Notes'" class="action-btn m-r-15"/>
+                            <GridLayout col="1" columns="auto,auto" @tap="toggleMenu" :dataId="n.id">
+                                <Label col="0" :text="_L('dismiss')" class="action-btn" />
+                                <Image col="1" src="~/images/Icon_Menu_Down.png" width="8" class="m-l-2" :class="isAndroid ? 'm-t-2' : 'm-t-8'"/>
+                            </GridLayout>
+                        </GridLayout>
+                    </GridLayout>
+                    <GridLayout rows="*,*" class="size-12 menu" horizontalAlignment="right" v-if="showMenu !== n.id">
+                        <Label row="0" :text="_L('notificationRemindLater')" textWrap="true" class="bold m-b-10" />
+                        <Label row="1" :text="_L('notificationDontRemind')" textWrap="true" class="bold" />
+                    </GridLayout>
                 </GridLayout>
             </StackLayout>
         </ScrollView>
     </Page>
 </template>
 <script lang="ts">
 import Vue from "vue";
+import {isAndroid, Label} from "@nativescript/core";
 export default Vue.extend({
     data() {
-        return {};
+        return {
+            showMenu: -1
+        };
+    },
+    props: {
+        notifications: {
+            type: Array
+        },
+    },
+    computed: {
+        isAndroid() {
+            return isAndroid;
+        },
+        activeNotifications() {
+            return this.notifications;
+        },
+        dismissedNotifications() {
+            return this.notifications;
+        }
     },
-    props: ["notifications"],
     methods: {
         onPageLoaded(this: any, args) {},
+        onLabelLoadedCentered(args) {
+            const lbl = args.object as Label;
+            if (isAndroid) {
+                lbl.android.setGravity(17);
+            }
+        },
+        onLabelLoadedVerticalCentered(args) {
+            const lbl = args.object as Label;
+            if (isAndroid) {
+                lbl.android.setGravity(16);
+            }
+        },
+        toggleMenu(this: any, event) {
+            this.showMenu = this.showMenu == event.object.dataId ? -1 : event.object.dataId;
+        },
         dismiss(this: any, event) {
             // Change color when pressed
-            let cn = event.object.className;
```

Comment:
There's a helper in animations.ts for doing this in a cleaner way. This was an old way and I'd love to find a better one, honestly, even the helper in animations is a little ugly IMO.